cql3/operation.cc

```diff
@@ -343,9 +343,9 @@ operation::element_deletion::affected_column() const {
 shared_ptr<operation>
 operation::element_deletion::prepare(data_dictionary::database db, const sstring& keyspace, const column_definition& receiver) const {
     if (!receiver.type->is_collection()) {
-        throw exceptions::invalid_request_exception(format("Invalid deletion operation for non collection column {}", receiver.name()));
+        throw exceptions::invalid_request_exception(format("Invalid deletion operation for non collection column {}", receiver.name_as_text()));
     } else if (!receiver.type->is_multi_cell()) {
-        throw exceptions::invalid_request_exception(format("Invalid deletion operation for frozen collection column {}", receiver.name()));
+        throw exceptions::invalid_request_exception(format("Invalid deletion operation for frozen collection column {}", receiver.name_as_text()));
```

Comment:
We should have a formatter for receiver that makes sense everywhere. But won't delay this fix.

Reply:
There are several ways to print "receiver" (the full operation? just the name) and receiver.name() indicates we want just the name. Or so thought the person who wrote this code which forgot (in a few places) the distinction between receiver.name( ) name_as_text().

What bothers me more is that we have silly formatters for "bytes" and "shared_ptr" - and I think that almost in no cases do we really want to print ASCII codes or a pointer - and almost every place we did this, it was unintentional. Personally I would have liked these formatters either to not exist or do something reasonable, and in the rare cases where somebody really wants to print a pointer or ASCII, we could have an explicit convertor for that (e.g., to_hex(bytes), or to_address(ptr)).

Reply:
formatters can have modifiers that say how you want it printed.

Reply:
I'm not a fan of that. C++ has methods, it's complicated enough I think it's good enough, we don't need to start adding "mixins" in the form of formatter modifiers for specific classes.